src/views/Onboarding/Sections/SmokefreePhase.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import withStyles from "@material-ui/core/styles/withStyles";
+
+const styles = theme => ({
+    icon: {
+        textDecoration: 'none',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        color: 'inherit',
+        padding: '0 10px',
+        maxWidth: 320,
+        margin: '0 0 20px',
+        '&:hover $iconBgImage': {
+            boxShadow: '0px 5px 10px 0px rgba(40, 40, 40, 0.1)',
+            transform: 'scale(1.05)',
```

Comment:
I think scaling on hover is not necassary for the circles

Reply:
Ok, done!